src/openrct2-ui/TextComposition.cpp

```diff
@@ -175,6 +175,13 @@ void TextComposition::HandleMessage(const SDL_Event* e)
                     CursorRight();
                     console.RefreshCaret();
                     break;
+                case SDLK_c:
+                    if (modifier & KEYBOARD_PRIMARY_MODIFIER && _session.Length)
```

Comment:
Just to be explicit about evaluation order here, I would suggest adding parentheses, as done in other such constructs around:
```suggestion
                    if ((modifier & KEYBOARD_PRIMARY_MODIFIER) && _session.Length)
```